content/agent/faq/how-do-i-uninstall-the-agent.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How do I uninstall the agent?
+kind: faq
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: Learn more about the Datadog Agent
+aliases:
+  - /agent/faq/how-do-i-uninstall-the-agent/
+---
+
+## Agent v6
+### Amazon Linux/Debian/Fedora/Ubuntu
+```
+$ sudo apt-get --purge remove datadog-agent -y
+```
+
+### CentOS
+
+* For CentOS 5:
+
+    ```
+    $ sudo yum remove datadog-agent-base
+    ```
+
+* For CentOS 6:
+
+    ```
+    $ sudo yum remove datadog-agent
+    ```
+
+### Mac OS
+
+Stop and Close the Datadog Agent: via the bone icon in the Tray.
```

Comment:
I would remove the capital C on `Close`

Reply:
What does the `Tray` refer to? Is that the top bar on the righthand corner? 

<img width="596" alt="screen shot 2018-03-01 at 11 03 29 am" src="https://user-images.githubusercontent.com/8942499/36854903-345bd116-1d40-11e8-97ae-12c10a0b6c7d.png">

Reply:
👌 yup indded :) the bone that you see in the tray is the DD agent